app/Http/Controllers/Three/PostsController.php

```diff
@@ -49,7 +49,6 @@ public function __construct(PostService $posts, SignupRepository $signups, PostT
         $this->transformer = $transformer;
 
         $this->middleware('auth:api', ['only' => ['store', 'update', 'destroy']]);
-        $this->middleware('role:admin', ['only' => ['store', 'update', 'destroy']]); // @TODO: Allow anyone to use this.
```

Comment:
Right now, I don't think we allow users to destroy their own posts. Should we keep this with only destroy in it? 

`$this->middleware('role:admin', ['only' => [destroy']]);`

Reply:
IMO, let's keep it for now?

Reply:
I agree! Updated :)